Sources/Corvus/Protocols/RestApi.swift

```diff
@@ -20,3 +20,20 @@ extension RestApi {
         content.register(to: routes)
     }
 }
+
+/// A convenience wrapper around `RestApi`.
+///
+/// Wraps `RestApi` by using a `Group` component
+/// to reduce indendation when creating a Api.
+public final class Api: RestApi {
```

Comment:
I know it's a small thing, but can you add comments to the public interfaces? :D